### ISSUE TITLE
[Adaptive Cards- Home- Flight update]: Talkback not announcing role and position of the menu/list items(Red, Green, blue).

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8447]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

**Test Environment**
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1

**Repro Steps:**

1. Turn on Talkback.
2. Open Adaptive Cards Visualizer Application.
3. Home Page will opened.
4. Now Navigate to 'Sample card' button and select it.
5. Now navigate to "choose sample" expands/collapse control and select "Inputs" list item.
6. Now Navigate to Visual tab and select it.
7. Now navigate to Inputs card using swipe gesture and expand "What color do you want?" dropdown button.
9. Now Navigate to menu items and observe the issue.

**Actual Result:**
Talkback not announcing role and position of the menu/list items((Red, Green, blue). Talkback announce like **"ticked, Red in list "**.

**Expected Result:**
Talkback should announce the role and the position of the menu/list items((Red, Green, blue). Talkback should announce like **"ticked, Red 1 of 3, list/menu item "**.

**User Experience:**
Knowing the role for a control helps users know what will happen when that object has focus and is selected, as well as what keyboard shortcuts to use when interacting with the control. When the component’s state/value/property is not defined, assistive technology users are often blocked from using the control or knowing what to expect when interacting with the component.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0)

### Screenshots

https://user-images.githubusercontent.com/89897257/230354375-911c9118-1a15-4e25-b17b-df11759a8e3b.mp4

### Card JSON

```JSON
NA
```

**Comment by VikrantSingh01** [comment]
@ardlank : Can you pls check once.
